.github/workflows/upstream-wpt-changes.yml

```diff
@@ -2,6 +2,8 @@ name: WPT export
 on:
   pull_request_target:
     types: ['opened', 'synchronize', 'reopened', 'edited', 'closed']
+    paths:
+      - 'tests/wpt/**'
```

Comment:
This is looking really cool. One comment on this line though: I think we cannot add this here. When the PR previously changed files in tests/wpt, but then doesn't it still needs to run in order to close the upstream PR. That's the reason I didn't add the 'paths' directive originally.

Reply:
Is that edge case happening frequently?

Reason for this change is that `WPT export` is run on many times for nothing. Maybe this edge case could be handled by bors or by additional scheduled workflow that would check for "stale" WPT export PRs.

Reply:
I think this adds a lot of complication when the proper functioning of the script requires that it always run. In general, it's very fast in comparison to the quick check and was before running for every PR, just on another server as a webhook. One thing we could do is to consider integrating it into the quick check script so that it doesn't show up at the bottom of the PR and also fixing the bug where it does not work well when encountering a merge commit.

Reply:
If we integrate it into quick-check, don't we then need to run quick-check for all those PR events?

Anyway if it isn't that simple I would just remove it and leave it to the future, as there is enough other stuff in this PR already.